src/main/scala/kinesis/mock/LoggingContext.scala

```diff
@@ -12,6 +12,15 @@ final case class LoggingContext private (context: Map[String, String]) {
   def addJson(key: String, js: Json): LoggingContext = copy(
     context + (key -> js.noSpacesSortKeys)
   )
+  def addEncoded[A: Encoder](
```

Comment:
New logging context function which will encode a value based on the new Encoder trait. It will be a more accurate printout of the data.